packages/tasks/src/tasks/text-to-3d/data.ts

```diff
@@ -0,0 +1,30 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "3D-Future is a dataset designed to generate 3D images from textual descriptions.",
+			id: "3D-Future",
```

Comment:
This dataset isn't on Hugging Face Hub, perhaps you can create the dataset? would be great apparently there's none on Hub and you'd be the first person to add such dataset!